subsys/logging/log_backend_native_posix.c

```diff
@@ -3,6 +3,7 @@
  * Copyright (c) 2018 Intel Corporation
  *
  * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileComment: IEC-61508-SIL3
```

Comment:
Tagging native_posix (or POSIX arch related) files would seem a mistake. (this is a test platform, so none of this code should end up in any embedded binary)

Reply:
Very likely mistakenly marked.

Reply:
Please see 

## Concerning this PR and the 800+ files changed

in the description.